lib/themes/dosomething/paraneue_dosomething/js/utilities/Analytics.js

```diff
@@ -42,6 +42,12 @@ function subscribeEvents() {
   });
 }
 
+export function sendEvent(category, action, label) {
+  if (typeof ga !== 'undefined' && ga !== null) {
+    ga('send', 'event', category, action, label);
+  }
+}
+
 function init() {
   // Only fire GA Custom Events if the GA object exists.
   if (typeof ga !== 'undefined' && ga !== null) {
```

Comment:
you could maybe add it as an exported function, something like `exists()` and it returns a boolean. You can call it in the `init()` and the `sendEvent()`. When exporting the `sendEvent()` I believe it'll grab it too, rather than repeating it in two places (code smells and all)...